freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_healthagent_setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+#
+# Add anonymous access only for replication agreements
+#
+
+export HOSTCN=`hostname -d| sed -E 's/^|\./,dc=/g' | sed 's/^,//g'`
+
+ldapsearch -x -h localhost -D "cn=directory manager" -w $FPW -b "$HOSTCN" -s base "(objectclass=*)"
+status=$?
```

Comment:
I just noticed that this duplicates the content of `freeipa_check_replication.sh`. could you update this to have it only in one place, eg. like a method and use that in both place?

Reply:
changed.